distributed/utils.py

```diff
@@ -289,6 +290,21 @@ def quiet():
     return results
 
 
+T = TypeVar("T")
+
+
+def asyncify(func: Callable[..., T]) -> Callable[..., Awaitable[T]]:
```

Comment:
In some places I've used an async wrapper to have a lighter footprint on the API, but more functions could just be made async in the first place.

Reply:
Yeah, I almost raised a question about this until I looked at how these methods were used.  I agree that making them sync by default is probably the right choice.  This seemed like a good solution to me.